lang/ilias_et.lang

```diff
@@ -1953,7 +1953,7 @@ chatroom_adm#:#client_settings#:#Üldised seaded
 chatroom_adm#:#enable_osd#:#Enable On-Screen notifications
 chatroom_adm#:#general_settings_title#:#Üldised jututoa seaded
 chatroom_adm#:#https#:#HTTPS
-chatroom_adm#:#osd_intervall#:#Refreshinterval (seconds)
+chatroom_adm#:#osd_interval#:#Refreshinterval (seconds)
```

Comment:
Please only touch the `de` and `en` language files. This change has to be reverted.